OutOfSchool/OutOfSchool.BusinessLogic/Services/CurrentUserService.cs

```diff
@@ -226,9 +224,10 @@ private async Task<bool> ProviderHasRights(Guid providerId)
         return result;
     }
 
+    // TODO: Need to check which method should stay after ProviderAdmin -> Employee refactoring.
     private async Task<bool> ProviderAdminHasRights(string providerAdminId)
     {
-        if (!this.IsEmployee() || UserId != providerAdminId)
+        if (!this.IsInRole(Role.Employee) || UserId != providerAdminId)
```

Comment:
Shouldn't it be `!InRole(Provider)`

Reply:
Ah, I've read your comment. "... we are going to have Employee and Position to check what an employee can do"
Now resolving this one